build/BUILD.bazel

```diff
@@ -44,6 +45,7 @@ go_test(
     ],
     embed = [":go_default_library"],
     deps = [
+        "//tables:go_default_library",
```

Comment:
Is it required? I don't see any usages of `tables` in the pull reqest.

Reply:
This is just from running gazel. I can manually remove it if it's a problem.